plugin_customizations/tribe_events/tribe_events_scripts.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+add_action('wp_enqueue_scripts', static function () {
+    $assetsUrl  = SDRT_FUNCTIONS_URL . 'assets/';
+    $assetsPath = SDRT_FUNCTIONS_DIR . '/assets/';
```

Comment:
Should we define these instead in the root file with other Globals? 
https://github.com/SDRefugeeTutoring/sdrt-custom-functions/blob/master/sdrt_custom_functions.php

Reply:
Honestly, I'm not quite as gung ho about constants as some folks. I can if you'd like, but otherwise I'm fine with this.